src/init.cpp

```diff
@@ -576,6 +580,7 @@ std::string LicenseInfo()
            "\n";
 }
 
+#if defined(HAVE_STD__SYSTEM) || defined(WIN32)
```

Comment:
I think repeating this condition everywhere is error-prone. Can't we make sure `HAVE_STD__SYSTEM` is always set when `WIN32` instead?

Reply:
I changed it to check for either `std::system` or `::wsystem`, so we can now use `HAVE_SYSTEM`.